test_io.py

```diff
@@ -9,7 +9,9 @@ def test_io_first_deck_file(monkeypatch, capsys):
     main()
     captured_stdout, _ = capsys.readouterr()
     output_text = captured_stdout.split("\n")
-    assert output_text[0] == "Please specify the deck you would like to load by typing in the file name:"
+    for line in output_text:
+        print(line)
+    assert output_text[0] == "Please specify the deck you would like to load by typing in the file name:", "Your output: " + output_text[0] + ' Expected: "Please specify the deck you would like to load by typing in the file name:" on line 0'
```

Comment:
My last comment disappeared into the void so I will put it hear again:

What happened to the if? I think the ifs make it a lot clearer to the user. 

For some reason when running pytest it will cut off long assertion statements, so it is hard to tell what the difference in the text was since these strings are so long. The if let us display the difference as verbose as possible. I am trying to find a way to have pytest display the entire assertion statements but it appears to be non trivial for new python users.

Reply:
Here is an example, I changed "name" to "nae" You can see it on my output but you cannot see what needs to change on the expected output. 

![image](https://user-images.githubusercontent.com/65638400/217863944-5dfd02c7-da83-4479-a3c4-17bd0a3827da.png)